tests/test_views.py

```diff
@@ -4622,5 +4568,277 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
 
+class TestDraftRegistrationViews(OsfTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestDraftRegistrationViews, self).setUp(*args, **kwargs)
+
+        self.user = AuthUserFactory()
+        self.node = ProjectFactory(creator=self.user)
+
+        MetaSchema.remove()
+        ensure_schemas()
+        self.meta_schema = MetaSchema.find_one(
+            Q('name', 'eq', 'Open-Ended Registration') &
+            Q('schema_version', 'eq', 1)
+        )
+        self.draft = DraftRegistrationFactory(
+            initiator=self.user,
+            branched_from=self.node,
+            registration_schema=self.meta_schema,
+            registration_metadata={
+                'summary': {'value': 'Some airy'}
+            }
+        )
+        self.draft.save()
+
+        current_month = datetime.datetime.now().strftime("%B")
+        current_year = datetime.datetime.now().strftime("%Y")
+
+        valid_date = datetime.datetime.now() + datetime.timedelta(days=180)
+        self.valid_embargo_payload = json.dumps({
+            u'embargoEndDate': unicode(valid_date.strftime('%a, %d, %B %Y %H:%M:%S')) + u' GMT',
+            u'registrationChoice': 'embargo',
+            u'summary': unicode(fake.sentence())
+        })
+        self.invalid_embargo_date_payload = json.dumps({
```

Comment:
Instead of dumping this to json, consider just using `self.app.post_json` where this data is used. (some tests already do this; might as well be consistent)